packages/plugin-commands-outdated/src/outdated.ts

```diff
@@ -76,6 +83,18 @@ export function help () {
             description: 'Prints the outdated packages in a list. Good for small consoles',
             name: '--no-table',
           },
+          {
+            description: 'Analyze only "dependencies" and "optionalDependencies"',
```

Comment:
I would suggest using "check" instead of "analyze" in the descriptions.

Reply:
Indeed, that sounds better